libs/libcommon/include/common/DateLUTImpl.h

```diff
@@ -283,15 +283,7 @@ class DateLUTImpl
     inline time_t toStartOfMinute(time_t t) const { return t / 60 * 60; }
     inline time_t toStartOfFiveMinute(time_t t) const { return t / 300 * 300; }
     inline time_t toStartOfFifteenMinutes(time_t t) const { return t / 900 * 900; }
-
-    inline time_t toStartOfTenMinutes(time_t t) const
-    {
-        if (offset_is_whole_number_of_hours_everytime)
-            return t / 600 * 600;
-
-        time_t date = find(t).date;
-        return date + (t - date) / 600 * 600;
-    }
+    inline time_t toStartOfTenMinutes(time_t t) const { return t / 600 * 600; }
```

Comment:
Fixed to be more consistent with other `toStartOf*Minutes()` functions and to produce equal results with `toStartOfInterval(t, INTERVAL 10 minute)`.